### PR TITLE
Update Document for thirdPartyCloud storage API

### PR DESCRIPTION
**Pull request by mahimaMSFT**
For more information about how to contribute to this repo, visit [this page](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md).

## Description

> Added more details in the document for the API.

> If this Pull Request should close/resolve any issues when merged, use [the special syntax for that](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) here.

### Main changes in the PR:
Added more details in the document.

### Validation performed:
Generated the document locally.

### Change file added:
packages\teams-js\src\public\thirdPartyCloudStorage.ts

> Ensure the change file meets the [formatting requirements](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md#change-log-using-beachball).

Yes

### CODE REVIEW

File: packages/teams-js/src/public/thirdPartyCloudStorage.ts [hunk 1]
```diff
@@ -124,10 +124,14 @@ export namespace thirdPartyCloudStorage {
 
   /**
    * Defines the callback function received from Third Party App
+   * @param {FilesFor3PStorage} files - array which will store all the files.
```
Comment (shrshindeMSFT) [inline packages/teams-js/src/public/thirdPartyCloudStorage.ts:127]: Does dis links to proper parameter in documentation?

File: packages/teams-js/src/public/thirdPartyCloudStorage.ts [hunk 2]
```diff
@@ -124,10 +124,14 @@ export namespace thirdPartyCloudStorage {
...
+   * @param {SdkError} error - sdk error
```
Comment (shrshindeMSFT) [inline packages/teams-js/src/public/thirdPartyCloudStorage.ts:128]: Same here. Try looking into `@link` from js/ts documentation.

File: packages/teams-js/src/public/thirdPartyCloudStorage.ts [hunk 3]
```diff
@@ -127,7 +127,11 @@ export namespace thirdPartyCloudStorage {
    * @beta
    */
   export interface DragAndDropFileCallback {
-    /** Callback from third party app */
+    /**
+     * Defination of the callback which is received from third party app when calling {@link thirdPartyCloudStorage.getDragAndDropFiles}
```
Comment (shrshindeMSFT) [inline packages/teams-js/src/public/thirdPartyCloudStorage.ts:131]:
```suggestion
     * Definition of the callback which is received from third party app when calling {@link thirdPartyCloudStorage.getDragAndDropFiles}
```

File: packages/teams-js/src/public/thirdPartyCloudStorage.ts [hunk 4]
```diff
@@ -127,7 +127,11 @@ export namespace thirdPartyCloudStorage {
...
+     * An array of dragdropped files {@link thirdPartyCloudStorage.FilesFor3PStorage}
```
Comment (shrshindeMSFT) [inline packages/teams-js/src/public/thirdPartyCloudStorage.ts:132]:
```suggestion
     * An array of drag and dropped files {@link thirdPartyCloudStorage.FilesFor3PStorage}
```

File: packages/teams-js/src/public/thirdPartyCloudStorage.ts [hunk 5]
```diff
@@ -136,7 +140,8 @@ export namespace thirdPartyCloudStorage {
   /**
    * Get drag-and-drop files using a callback.
    *
-   * @param {string} dragAndDropInput - Teams thread id or Teams conversation id from a Teams chat/channel
+   * @param {string} dragAndDropInput - unique id which is a combination of replyToId + threadId of teams chat and channel.
+   *   Both ReplyToId and threadId can be fetched from application context.
```
Comment (shrshindeMSFT) [inline packages/teams-js/src/public/thirdPartyCloudStorage.ts:144]:
```suggestion
   *   Both replyToId and threadId can be fetched from application context.
```

File: packages/teams-js/src/public/thirdPartyCloudStorage.ts (hunk 3 above)
Comment (mahimaMSFT) [reply packages/teams-js/src/public/thirdPartyCloudStorage.ts:131]: acknowledged

File: packages/teams-js/src/public/thirdPartyCloudStorage.ts (hunk 4 above)
Comment (mahimaMSFT) [reply packages/teams-js/src/public/thirdPartyCloudStorage.ts:132]: acknowledged

File: packages/teams-js/src/public/thirdPartyCloudStorage.ts (hunk 5 above)
Comment (mahimaMSFT) [reply packages/teams-js/src/public/thirdPartyCloudStorage.ts:144]: acknowledged

**Review by shrshindeMSFT — DISMISSED** [review]
LGTM 🚢

File: change/@microsoft-teams-js-ab962b64-97fb-4ebd-93de-c638eb3dd8eb.json [hunk 6]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Updated documentation for `{namespace}` capability.",
```
Comment (TrevorJoelHarris) [inline change/@microsoft-teams-js-ab962b64-97fb-4ebd-93de-c638eb3dd8eb.json:3]:
```suggestion
  "comment": "Updated documentation for `thirdPartyCloudStorage` capability.",
```

**Review by TrevorJoelHarris — APPROVED** [review]
Changefile needs an update, looks good otherwise

**Review by shrshindeMSFT — APPROVED** [review]

**Pull request merged**